src/app.js

```diff
@@ -1 +1,46 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+async function fetchCountries() {
```

Comment:
Ik merk dat fetchCountries meer doet dan alleen de countries fetchen. in kleine projectjes zal je het niet heel snel merken. Maar voor de grotere projecten wil je echt dat je functies maar 1 ding doen. Als je nu nog eens gewoon de data wil van de country API for een nieuwe functie moet je hier een andere functie voor schrijven of opnieuw de code schrijven om het aan te roepen.